packages/elements/src/datetime-field/__test__/datetime-field.navigation.test.js

```diff
@@ -1,4 +1,4 @@
-import { fixture, expect, elementUpdated, nextFrame } from '@refinitiv-ui/test-helpers';
+import { fixture, expect, elementUpdated, nextFrame, isSafari } from '@refinitiv-ui/test-helpers';
```

Comment:
I don't think we should add to test-helpers because it is only used in datetime-field.

Reply:
Updated